ios/RNOneginiSdk.swift

```diff
@@ -308,8 +308,14 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
             return
         }
 
-        let completion = makeCompletionHandler(resolver: resolve, rejecter: reject, resolveWithNil: false)
-        bridgeConnector.toResourceHandler.resourceRequest(resourceType, details, completion)
+        bridgeConnector.toResourceHandler.resourceRequest(resourceType, details) { result in
+            switch result {
+            case .success(let response):
+                resolve(response.asDictionary)
+            case .failure(let error):
+                self.rejectWithError(reject, error)
```

Comment:
Are you sure it can be strong reference here?

Reply:
yes, as this is the entry point of the RN wrapper that should be fine.